apps/site/lib/site_web/controllers/project_controller.ex

```diff
@@ -218,6 +219,9 @@ defmodule SiteWeb.ProjectController do
 
   @spec simplify_teaser(map()) :: map()
   defp simplify_teaser(teaser) do
-    Map.take(teaser, ~w(id image path title routes date status)a)
+    teaser
+    |> Map.put(:path, project_path(SiteWeb.Endpoint, :show, teaser))
+    |> Map.put(:formatted_date, ViewHelpers.format_full_date(teaser.date))
```

Comment:
In `projects/components/MoreProjectsRow.tsx` there's an existing `formattedDate` function that you could pull out into a helper and re-use, rather than formatting the date on the backend and passing it in.